pattern-library/sass/patterns/_forms.scss

```diff
@@ -257,6 +257,14 @@ $input-width: (
             border-color: palette(error, text);
         }
     }
+
+    // Input fields immediately followed by a button should eliminate the spacing between them
```

Comment:
I'm not sure, but I wonder if this will be too general a rule. Having said that, I'm trying to think of a situation where you might need a button afterwards that is not associated with the input, but I haven't thought of one. It is just that I was expecting this rule to be associated with a class like ``search-form``.